stripe/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsActivity.kt

```diff
@@ -116,12 +113,13 @@ internal class PaymentOptionsActivity : BaseSheetActivity<PaymentOptionResult>()
                 PaymentOptionResult.Failed(it)
             )
         }
-        bottomSheetController.shouldFinish.observe(this) { shouldFinish ->
-            if (shouldFinish) {
-                finish()
-            }
-        }
-        bottomSheetController.setup()
+
+        // Fragment is added with animation immediately after activity is created, so the sheet can
+        // be expanded now and it'll animate from the bottom.
+        bottomSheetController.expand()
+        // isFitToContents causes conflicts when calculating the sheet position upon resize.
+        // CoordinatorLayout will already position the sheet correctly with gravity = bottom.
+        bottomSheetBehavior.isFitToContents = false
```

Comment:
isFitToContents is also set to false in the case of hidden.   Is there a way to keep all the setting of that field inside the BottomSheetController?